### DISCUSSION TITLE
Python versions in testing

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#301] [Testing] [answered]
We currently test against 3.6, 3.7 and 3.8. 3.6 is no longer supported so I suggest we drop this. Versions I'd recommend people use (all things being equal) are 3.9 and 3.10 as these are getting bug fixes. Maybe add these in?

https://www.python.org/downloads/

**Comment by haipinglu** [comment, marked as answer]
Thanks Bob! Yes, I fully agree. Could you please update the tests (together with other fixes)?
